.coveragerc

```diff
@@ -1,4 +1,5 @@
 [run]
+concurrency = multiprocessing,thread
```

Comment:
Add concurrency = multiprocessing, so that codecov can count the coverage of subprocesses, and the default concurrency is set to threading. However，there are some things need to pay attention in using, refer to: https://pytest-cov.readthedocs.io/en/latest/subprocess-support.html